classes/submission/form/SubmissionSubmitStep1Form.php

```diff
@@ -125,6 +126,30 @@ public function setSubmissionData($submission)
         $submission->setSeriesId($this->getData('seriesId'));
         parent::setSubmissionData($submission);
     }
+
+    /**
+     * Save changes to submission.
+     * @return int the submission ID
+     */
+	function execute(...$functionParams) {
+		
+		$submissionId = parent::execute(...$functionParams);
+
+		$request = Application::get()->getRequest();
+
+        $seriesDao = DAORegistry::getDAO('SeriesDAO'); /** @var SeriesDAO $seriesDao */
```

Comment:
Check your editor; it looks like the indentation is inconsistent. (The code formatter is supposed to run over this code before committing; I wonder if that's not working properly? It's configured using Husky for pre-commit hooks; see `package.json` for details.)

Reply:
fixed . very weird that my editor sometimes changing the indentation form `tab` to `spaces` but only to few files , not all .